dev-util/diffoscope/diffoscope-177.ebuild

```diff
@@ -17,9 +17,9 @@ LICENSE="GPL-3+"
 SLOT="0"
 KEYWORDS="~amd64 ~ppc64 ~x86"
 IUSE="acl binutils bzip2 libcaca colord cpio +diff docx dtc e2fsprogs file
```

Comment:
Honestly, this needs an optfeature conversion, but let's do this for now.

Reply:
Agreed, `optfeature` would be even better.